scripts/generate_mappings.py

```diff
@@ -0,0 +1,97 @@
+import json
+from collections import defaultdict
+from ontology_mapper.loader import NugetPackage
+from ontology_mapper.engine import MappingEngine
+
+with NugetPackage('mapped.ontologies.core.dtdl') as package:
+    mapped_ontology = package.get_content('1.23.8')
```

Comment:
Read this from an argument so that we can specify the version when you run it

Reply:
Can we meet to discuss this?

I wrote the [NugetPackage](https://github.com/mapped/ontology-mapper/blob/1b0d953d5d82833c74261477adf3985820c06b38/ontology_mapper/loader.py#L9) so that is doesn't actually retrieve the package until it run it executes `_download_package` method. 

I created a higher level function which retrieves the content of the most recent, or specified version:
https://github.com/mapped/ontology-mapper/blob/1b0d953d5d82833c74261477adf3985820c06b38/ontology_mapper/loader.py#L78

Reply:
I deliberately chose not to enforce the version to be specified, which will ensure that the ontology mapper is downloading the most recent versions of our ontologies. I provided a way to specify the version for scenarios where we wan't to prevent the most recent version from being downloaded.